olp-cpp-sdk-core/include/olp/core/client/OlpClient.h

```diff
@@ -38,50 +38,59 @@ class NetworkConfig;
 
 namespace client {
 /**
- * @brief This class is responsible for executing the REST requests.
+ * @brief Executes the REST requests.
```

Comment:
```suggestion
 * @brief Executes HTTP request.
```
Not related to the change, as the previous version of the docs was not entirely correct. Would that read a bit simpler and cleaner if it were reading that it executes [any] HTTP requests?

Reply:
Changed